src/makeCalculator.js

```diff
@@ -38,6 +38,47 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
```

Comment:
let's use `this` instead of `calculator`, that would be more consistent
you'll hit into the issues with context loss, so here is the relevant article on this problem
https://www.freecodecamp.org/news/what-to-do-when-this-loses-context-f09664af076f/

Reply:
Implemented.